task/rpm-ostree/0.1/rpm-ostree.yaml

```diff
@@ -0,0 +1,230 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: image-build, appstudio, hacbs
+  labels:
+    app.kubernetes.io/version: "0.1"
+    build.appstudio.redhat.com/build_type: rpm-ostree
+    build.appstudio.redhat.com/multi-platform-required: "true"
+  name: rpm-ostree
+spec:
+  description: RPM Ostree
+  params:
+  - description: Reference of the image rpm-ostree will produce.
+    name: IMAGE
+    type: string
+  - default: quay.io/sdouglas/rpm-ostree:dev
+    description: The location of the rpm-ostree builder image.
+    name: BUILDER_IMAGE
+    type: string
+  - default: .
+    description: Path to the directory to use as context.
+    name: CONTEXT
+    type: string
+  - description: The file to use to build the image
+    name: IMAGE_FILE
+    type: string
+  - default: "true"
+    description: Verify the TLS on the registry endpoint (for push/pull to a non-TLS
+      registry)
+    name: TLSVERIFY
+    type: string
+  - default: ""
+    description: Delete image tag after specified time. Empty means to keep the image
+      tag. Time values could be something like 1h, 2d, 3w for hours, days, and weeks,
+      respectively.
+    name: IMAGE_EXPIRES_AFTER
+    type: string
+  - default: ""
+    description: The image is built from this commit.
+    name: COMMIT_SHA
+    type: string
+  - description: The platform to build on
+    name: PLATFORM
+    type: string
+  results:
+  - description: Digest of the image just built
+    name: IMAGE_DIGEST
+  - description: Image repository where the built image was pushed
+    name: IMAGE_URL
+  - description: Digests of the base images used for build
+    name: BASE_IMAGES_DIGESTS
+  - description: The counting of Java components by publisher in JSON format
+    name: SBOM_JAVA_COMPONENTS_COUNT
+    type: string
+  - description: The Java dependencies that came from community sources such as Maven
+      central.
+    name: JAVA_COMMUNITY_DEPENDENCIES
+  stepTemplate:
+    env:
+    - name: CONTEXT
+      value: $(params.CONTEXT)
+    - name: IMAGE_FILE
+      value: $(params.IMAGE_FILE)
+    - name: IMAGE
+      value: $(params.IMAGE)
+    - name: TLSVERIFY
+      value: $(params.TLSVERIFY)
+    - name: IMAGE_EXPIRES_AFTER
+      value: $(params.IMAGE_EXPIRES_AFTER)
+    - name: BUILDER_IMAGE
+      value: $(params.BUILDER_IMAGE)
+    name: ""
+    resources: {}
+  steps:
+  - env:
+    - name: COMMIT_SHA
+      value: $(params.COMMIT_SHA)
+    image: quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f
+    imagePullPolicy: Always
+    name: build
+    resources:
+      limits:
+        cpu: "2"
+        memory: 4402341478400m
```

Comment:
If I'm reading it right, this is about 4 Gi? Could it be specified in a more readable way?

Reply:
This was copied from the buildah-remote task originally, as I thought I could mostly keep them in sync. They are two different and this does not need 4g anyway, I will update.